tools/knb/pkg/plugin/plugin.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import "github.com/spf13/cobra"
+
+// Plugin represents plugin configuration data struct
+type Plugin struct {
+	Name             string
```

Comment:
Why not be explicit about the YAML annotation so it’s clear. Things like `PluginImportPath` make it hard to guess what the YAML field is?

Reply:
Fixed in https://github.com/knative/client/pull/1226/commits/d940570529633941f4fc8e88d98bea95f35d3b14.